srcpkgs/gtk-vnc/template

```diff
@@ -1,106 +1,46 @@
 # Template file for 'gtk-vnc'
 pkgname=gtk-vnc
-version=0.9.0
-revision=3
+version=1.0.0
+revision=1
+build_style=meson
 build_helper="gir"
-build_style=gnu-configure
-configure_args="$(vopt_enable gir introspection) $(vopt_enable vala)"
-hostmakedepends="automake gettext-devel glib-devel intltool libgcrypt-devel
- libtool pkg-config gobject-introspection $(vopt_if vala vala)"
-makedepends="gnutls-devel gtk+-devel gtk+3-devel libsasl-devel
- $(vopt_if vala 'vala')"
+configure_args="-Dwith-coroutine=gthread -Dwith-vala=$(vopt_if vala true false)"
+hostmakedepends="pkg-config"
+makedepends="gnutls-devel gtk+3-devel cyrus-sasl libgpg-error-devel expat-devel
+ perl-Text-CSV glib-devel libgcrypt-devel $(vopt_if vala vala)"
 short_desc="VNC viewer widget for GTK"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://wiki.gnome.org/Projects/gtk-vnc"
 distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
-checksum=3a9a88426809a5df2c14353cd9839b8c8163438cb708b31d8048c79d180fcab7
+checksum=a81a1f1a79ad4618027628ffac27d3391524c063d9411c7a36a5ec3380e6c080
 
 build_options="gir vala"
 build_options_default="gir vala"
 
-post_extract() {
-	cp -a ${wrksrc} /tmp/gtk2-build
-	cp -a ${wrksrc} /tmp/gtk3-build
-	rm -r ${wrksrc}/*
-	mv /tmp/gtk[23]-build ${wrksrc}
-}
-pre_configure() {
-	local build
-	if [ "$CROSS_BUILD" ]; then
-		find . -name "Makefile.*" -exec sed -i "{}" \
-			-e"s;\(\$(CODEGENDIR)\);${XBPS_CROSS_BASE}/\1;" \
-			-e"s;\(\$(DEFSDIR)\);${XBPS_CROSS_BASE}/\1;" \
-			\;
-	fi
-	for build in gtk2-build gtk3-build; do
-		cd ${wrksrc}/${build}
-		autoreconf -if
-	done
-}
-
-do_configure() {
-	cd ${wrksrc}/gtk2-build
-	./configure ${configure_args} --with-gtk=2.0
-
-	cd ${wrksrc}/gtk3-build
-	./configure ${configure_args} --with-gtk=3.0
-}
-
-do_build() {
-	cd ${wrksrc}/gtk2-build
-	make ${makejobs}
-
-	cd ${wrksrc}/gtk3-build
-	make ${makejobs}
-}
-
-do_install() {
-	cd ${wrksrc}/gtk2-build
-	make DESTDIR=${DESTDIR} install
-
-	cd ${wrksrc}/gtk3-build
-	make DESTDIR=${DESTDIR} install
-}
-
-gtk2-vnc_package() {
-	short_desc+=" - GTK+2 port"
-	pkg_install() {
-		vmove "usr/lib/libgtk-vnc-1.0.so*"
-		if [ "$build_option_gir" ]; then
-			vmove usr/lib/girepository-1.0/GtkVnc-1.0.typelib
-		fi
-	}
-}
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" perl glib-devel vala"
```

Comment:
merge into original hostmakedepends,
on native build, they're required by some packages in makedepends.

We don't need this line.